das/daser.go

```diff
@@ -242,7 +242,15 @@ func (d *DASer) sampleHeader(ctx context.Context, h *header.ExtendedHeader) erro
 		if err == context.Canceled {
 			return nil
 		}
-		d.checkForByzantineError(d.ctx, h.Hash(), uint64(h.Height), err)
+		var byzantineErr *ipld.ErrByzantine
+		if errors.As(err, &byzantineErr) {
+			sendErr := d.bcast.Broadcast(ctx, fraud.CreateBadEncodingProof(h.Hash(), uint64(h.Height), byzantineErr))
+			if sendErr != nil {
+				log.Errorw("fraud proof propagating failed", "err", err)
+			} else {
+				log.Warn("Shutting down services…")
+			}
```

Comment:
```suggestion
                         log.Warn("Bad encoding fraud detected, propagating proof...")
			sendErr := d.bcast.Broadcast(ctx, fraud.CreateBadEncodingProof(h.Hash(), uint64(h.Height), byzantineErr))
			if sendErr != nil {
				log.Errorw("fraud proof propagating failed", "err", err)
			} 
			log.Warn("Shutting down services…")

```

Reply:
I'd suggest to leave only `log.Warn("Propagating proof...")`

Reply:
If we are talking about logs here. I don't think "Shutting down services…" log is appropriate to be in DASer, while it should be either in the OnBefp or FraudService, or just every Service should log that it is shutting down on Stop

Reply:
agree. please take a look https://github.com/celestiaorg/celestia-node/pull/820/commits/f38abcac485c7ada59d9358de7f9e4630b0bd411